app/elements/submission/tasks/canvas-game-task.html

```diff
@@ -0,0 +1,350 @@
+<link rel="import" href="/bower_components/iron-icons/av-icons.html">
+<link rel="import" href="/bower_components/paper-button/paper-button.html">
+<link rel="import" href="/bower_components/paper-material/paper-material.html">
+<link rel="import" href="/elements/code-editor/code-editor.html">
+<link rel="import" href="/elements/code-editor/language-selector.html">
+<link rel="import" href="task-display.html">
+
+<dom-module id="canvas-game-task">
+	<style>
+		:host{
+			display: block;
+			height: 100%;
+		}
+		#canvas{
+			border: 1px solid black;
+			margin: 40px;
+			background: white;
+		}
+		#componentWrapper{
+			height: 100%;
+		}
+		#componentWrapper>div{
+			height: 100%;
+		}
+		paper-material{
+			width: 100%;
+			margin: auto;
+			background: white;
+		}
+		#codeEditorWrapper{
+			height: 100%;
+			width: 100%;
+		}
+		#leftCard{
+			margin: 16px 8px 16px 16px;
+			width: calc(100% - 24px);
+			height: calc(100% - 32px);
+			overflow: auto;
+		}
+		#topRightCard>h2{
+			height: 40px;
+			margin: 0;
+			padding-left: 16px;
+		}
+		#topRightCard{
+			overflow: auto;
+			margin: 16px 16px 16px 8px;
+			width: calc(100% - 24px);
+			height: calc(50% - 24px);
+		}
+		#botRightCard{
+			overflow: hidden;
+			margin: 8px 16px 16px 8px;
+			width: calc(100% - 24px);
+			height: calc(50% - 24px);
+		}
+		#buttons{
+			height: 45px;
+			margin: 16px 16px 16px 8px;
+		}
+		#consoleWrapper{
+			margin: 8px 16px 16px 16px;
+			height: calc(100% - 90px);
+		}
+		#buttons>paper-icon-button{
+			padding: 5px;
+			margin: 0;
+		}
+	</style>
+	<template>
+		<iron-ajax
+			id="ajaxRequest"
+			handle-as="json"
+			debounce-duration="1000"
+			content-type="application/json"
+			>
+		</iron-ajax>
+		<iron-ajax
+			id="getTemplateAjax"
+			handle-as="text"
+			on-response="onTemplateResponse"
+			on-error="onTemplateError"
+			>
+		</iron-ajax>
+		<div id="componentWrapper" class="horizontal layout">
+			<div class="flex">
+				<paper-material id="leftCard" elevation="1">
+					<canvas id="canvas" width="{{canvasWidth}}" height="{{canvasHeight}}"></canvas>
+				</paper-material>
+			</div>
+			<div class="flex">
+				<paper-material id="topRightCard" elevation="1">
+					<div id="codeEditorWrapper">
+						<code-editor id="codeEditor"></code-editor>
+					</div>
+				</paper-material>
+				<paper-material id="botRightCard" elevation="1">
+					<div id="buttons" class="horizontal layout">
+						<paper-icon-button icon="av:play-arrow" on-click="runCode">Run code</paper-icon-button>
+						<span class="flex"></span>
+						<paper-button raised noink class$="{{finishedButtonClass}}" on-click="finishTask">Finish</paper-button>
+					</div>
+					<div id="consoleWrapper">
+						 <task-display task="{{task}}"></task-display>
+					</div>
+				</paper-material>
+			</div>
+		</div>
+	</template>
+	<script>
+		Polymer({
+			// TODO: Extract canvas into own component
+			is: 'canvas-game-task',
+			properties: {
```

Comment:
you'd save a lot of lines if instead of 
`name : { 
    type: Something 
} `
you'd just write `name: Something`